ansible/roles/linux-perf/tasks/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+
+#
+# installs Linux perf @ /usr/local/bin/perf
+#
+
+- stat:
+    path=/usr/local/bin/perf
+  register: linux_perf
+  tags: linux_perf
+
+- name: install perf on linux
+  tags: linux_perf
+  when: not (linux_perf.stat.exists | bool)
+  block:
+    - name: Clean path
+      file:
+        state: absent
+        path: /data/tmp/
```

Comment:
Might be worth it to move `/data/tmp` into its own variable considering that it's used in 5 tasks.